PubSub/src/Subscription.php

```diff
@@ -351,6 +351,36 @@ public function detached(array $options = [])
      *           be between 0 and 600 seconds. Defaults to 600 seconds.
      *     @type bool $enableExactlyOnceDelivery Indicates whether to enable
      *           'Exactly Once Delivery' on the subscription.
+     *     @type array cloudStorageConfig If delivery to Google Cloud Storage is used
+     *           with this subscription, this field is used to configure it.
+     *     @type string $cloudStorageConfig.bucket User-provided name for the Cloud Storage bucket.
+     *           The bucket must be created by the user. The bucket name must be without
+     *           any prefix like "gs://". See the [bucket naming
+     *           requirements] (https://cloud.google.com/storage/docs/buckets#naming).
+     *     @type string $cloudStorageConfig.filenamePrefix
+     *           User-provided prefix for Cloud Storage filename. See the [object naming
+     *           requirements](https://cloud.google.com/storage/docs/objects#naming).
+     *     @type string $cloudStorageConfig.filenameSuffix
+     *           User-provided suffix for Cloud Storage filename. See the [object naming
+     *           requirements](https://cloud.google.com/storage/docs/objects#naming). Must
+     *           not end in "/".
+     *     @type array $cloudStorageConfig.textConfig If set, message data will be written
+     *           to Cloud Storage in text format.
+     *     @type array $cloudStorageConfig.avroConfig If set, message data will be written to
+     *           Cloud Storage in Avro format.
+     *     @type bool $cloudStorageConfig.avroConfig.writeMetadata
+     *           When true, write the subscription name, message_id, publish_time,
+     *           attributes, and ordering_key as additional fields in the output.
+     *     @type Duration|string $cloudStorageConfig.maxDuration The maximum duration
+     *           that can elapse before a new Cloud Storage file is created.
+     *           Min 1 minute, max 10 minutes, default 5 minutes. May not exceed the
+     *           subscription's acknowledgement deadline. If a string is provided,
+     *           it should be as a duration in seconds with up to nine fractional digits,
+     *           terminated by 's', e.g "3.5s"
+     *     @type int|string $cloudStorageConfig.maxBytes The maximum bytes that can be
```

Comment:
this should be `int`

Reply:
lets say user has to pass `max 10 GiB`, then they will pass (10^9), right?

Reply:
Yeah, but as string also they can pass. Say `"1000"`

Reply:
In generated class also they mentioned the same, ref: https://github.com/googleapis/google-cloud-php/blob/main/PubSub/src/V1/CloudStorageConfig.php#L93

Reply:
So, both formats seem to be accepted. The int (in bytes) way and the string way.